packages/core/strapi/lib/factories.d.ts

```diff
@@ -0,0 +1,46 @@
+import { Service } from './core-api/service';
+import { Controller } from './core-api/controller';
+
+type ControllerConfig = Controller;
+
+type ServiceConfig = Service;
+
+type HandlerConfig = {
+  auth: false | { scope: string[] };
+  policies: string[];
```

Comment:
I thought we can pass an inline function or an object as a policy or a middleware too?

Reply:
Indeed I'll update